src/main/java/io/swagslash/gametrackerserver/dto/webapp/GameDTO.java

```diff
@@ -0,0 +1,31 @@
+package io.swagslash.gametrackerserver.dto.webapp;
+
+/**
+ * DTO for delivering data to Frontend
+ */
+public class GameDTO {
```

Comment:
This should include gamemodes and genres?

Reply:
There is no need to add setters if this class is just used to deliver data to the frontend

Reply:
Gamemodes and genres will be added. Didn't exist in the IGDB Wrapper when PR was created.

Reply:
Getter and Setter are needed for proper Jackson serialzation